interpreter/cling/tools/packaging/cpt.py

```diff
@@ -590,6 +599,8 @@ def compile_for_binary(arg):
     global prefix, EXTRA_CMAKE_FLAGS
     prefix = arg
 
+    EXTRA_CMAKE_FLAGS = args.get('with_cmake_flags')
```

Comment:
Same here.

Reply:
I'll remove that that thank you for letting me know

Reply:
Yeah just removed both references